Source/Widgets/Animation/Animation.js

```diff
@@ -698,6 +703,31 @@ define([
      * animation.applyThemeChanges();
      */
     Animation.prototype.applyThemeChanges = function() {
+        // Since we rely on computed styles for themeing, we can't actually
+        // do anything if the container has not yet been added to the DOM.
+        // Set up an observer to be notified when it is added and apply
+        // the changes at that time.
+        if (!document.body.contains(this._container)) {
+            if (defined(this._observer)) {
+                //Already listening.
+                return;
+            }
+            var that = this;
+            that._observer = new MutationObserver(function(mutations) {
+                mutations.forEach(function(mutation) {
+                    for (var i = 0; i < mutation.addedNodes.length; i++) {
+                        if (mutation.addedNodes[i].contains(that._container)) {
```

Comment:
This is what I got: 
![image](https://cloud.githubusercontent.com/assets/3451886/25811544/a2b4fa96-33e1-11e7-989f-7ef141597447.png)